From allisonrandal@gmail.com  Fri Oct  2 15:00:25 2009
From: Allison Randal <allisonrandal@gmail.com>
Date: Thu, 24 Sep 2009 00:19:43 -0700
Subject: Staging: comdi: ni_at_ao.c: fix coding style error
To: Greg Kroah-Hartman <gregkh@suse.de>
Cc: Allison Randal <allison@parrot.org>
Message-ID: <1253776783-17367-1-git-send-email-allisonrandal@gmail.com>


From: Allison Randal <allison@parrot.org>

The line was too long, used braces on single line for loop body.

Signed-off-by: Allison Randal <allison@parrot.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/comedi/drivers/ni_at_ao.c |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

--- a/drivers/staging/comedi/drivers/ni_at_ao.c
+++ b/drivers/staging/comedi/drivers/ni_at_ao.c
@@ -32,7 +32,8 @@ Configuration options:
   [0] - I/O port base address
   [1] - IRQ (unused)
   [2] - DMA (unused)
-  [3] - analog output range, set by jumpers on hardware (0 for -10 to 10V bipolar, 1 for 0V to 10V unipolar)
+  [3] - analog output range, set by jumpers on hardware (0 for -10 to 10V
+	bipolar, 1 for 0V to 10V unipolar)
 
 */
 /*
@@ -431,9 +432,8 @@ static int atao_calib_insn_read(struct c
 				struct comedi_insn *insn, unsigned int *data)
 {
 	int i;
-	for (i = 0; i < insn->n; i++) {
+	for (i = 0; i < insn->n; i++)
 		data[i] = 0;	/* XXX */
-	}
 	return insn->n;
 }
 
